jbpm-services/jbpm-services-api/src/build/revapi-config.json

```diff
@@ -45,7 +45,28 @@
           "methodName": "getTimers",
           "elementKind": "method",
           "justification": "JBPM-7632 - Enhance process definition data including timers and nodes"
-        }
+        },
+         {
+         "code": "java.method.addedToInterface",
+         "new": "method java.util.List<org.kie.internal.runtime.error.ExecutionError> org.jbpm.services.api.admin.ProcessInstanceAdminService::getErrorsByDeploymentId(java.lang.String, boolean, org.kie.api.runtime.query.QueryContext)",
+         "package": "org.jbpm.services.api.admin",
+         "classSimpleName": "ProcessInstanceAdminService",
+         "methodName": "getErrorsByDeploymentId",
+         "elementKind": "method",
+         "justification": "Added support for filtering errors by deployment id"
+         },
```

Comment:
Is this needed as well? You didn't do any changes related to ProcessInstanceAdminService...

Reply:
Done